src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -97,7 +97,7 @@ static int
 fd_double_is_normal( double dbl ) {
   ulong x = fd_dblbits( dbl );
   int is_denorm =
-    ( fd_dblbits_bexp( x ) == 0 ) |
+    ( fd_dblbits_bexp( x ) == 0 ) &
     ( fd_dblbits_mant( x ) != 0 );
```

Comment:
This function seems like it would mistakenly classify 0 as a normal number. Could you please double check that? If so then this line can be removed, or we should add another case that explicitly checks for 0 (for neatness)

Reply:
@anarcheuz it would be good to add a comment here explaining that we are explicitly allowing zero, and the reason.

Reply:
Based on my reading of what I find on the Internet, 0 is not a subnormal number